src/main/java/com/fleotadezuta/youthprogrammanager/persistence/repository/TicketRepository.java

```diff
@@ -3,7 +3,9 @@
 import com.fleotadezuta.youthprogrammanager.persistence.document.TicketDocument;
 import org.springframework.data.mongodb.repository.ReactiveMongoRepository;
 import org.springframework.stereotype.Repository;
+import reactor.core.publisher.Flux;
 
 @Repository
 public interface TicketRepository extends ReactiveMongoRepository<TicketDocument, String> {
+    Flux<TicketDocument> findAllByChildId(String userId);
```

Comment:
Egyfelől a paraméter neve nem jó, másfelől nem vagyok róla meggyőződve, hogy nincs ilyen generált metódusa a TicketRepository-nak egyébként is (anélkül, hogy itt explicit ki lenne írva)